frontend/public/components/edit-yaml.jsx

```diff
@@ -14,10 +15,14 @@ import 'brace/snippets/yaml';
 import { k8sCreate, k8sUpdate, referenceFor, getCompletions, groupVersionFor, snippets, referenceForModel } from '../module/k8s';
 import { history, Loading, resourceObjPath } from './utils';
 import { SafetyFirst } from './safety-first';
-import { coFetchJSON } from '../co-fetch';
 import { ResourceSidebar } from './sidebars/resource-sidebar';
 import { yamlTemplates } from '../models/yaml-templates';
 
+import { NativeTypes } from 'react-dnd-html5-backend';
```

Comment:
Lets import the 'react-dnd-html5-backend' and 'react-dnd' in the first import block since those are 3rd party components and `'./utils/drag-drop-context'` in the last import block which are our components.